### DISCUSSION TITLE
Welcome to shillelagh Discussions!

### DISCUSSION THREAD

**Discussion by betodealmeida** [betodealmeida/shillelagh#398] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by blackerby** [comment]
Greetings! My name is William and I learned of Shillelagh through the Apache Superset project. I'm using @betodealmeida's great docs to learn about creating custom adapters. I'm hoping to create and maintain at least one custom third party adapter. Here's what I'm working on now: https://github.com/blackerby/shillelagh-govinfo. It's in very early stages of development and I'm using the project to teach myself about modern open source Python development practices, so it's not perfect but I'm having a great time working on it.

**Comment by betodealmeida** [comment]
> I'm hoping to create and maintain at least one custom third party adapter.

Awesome! Don't hesitate to ask if you have any questions or if something's not clear!
